e2e-tests/fixtures/SequenceEditor.ts

```diff
@@ -0,0 +1,144 @@
+import { Locator, Page, expect } from '@playwright/test';
+import { readFileSync } from 'fs';
+import { adjectives, animals, colors, uniqueNamesGenerator } from 'unique-names-generator';
+
+export class SequenceEditor {
+  command: Locator;
+  detailPane: Locator;
+  editor: Locator;
+  jsonEditor: Locator;
+  jsonImport: Locator;
+  jsonPath: string = 'e2e-tests/data/ban00001.json';
+  linter: Locator;
+  parcel: Locator;
+  sequenceName: string;
+  sequenceNameTextbox: Locator;
+  tableRow: Locator;
+
+  constructor(public page: Page) {}
+
+  async createSequence(parcelName: string): Promise<string> {
+    await this.page.locator('button:has-text("New")').click();
+    await this.page.waitForURL('/sequencing/new');
+
+    // select parcel
+    this.updatePage(this.page);
+    await this.parcel.selectOption(parcelName);
+
+    // name sequence
+    await this.sequenceNameTextbox.click();
+    this.sequenceName = uniqueNamesGenerator({ dictionaries: [adjectives, colors, animals] });
+    await this.sequenceNameTextbox.fill(this.sequenceName);
+    await this.sequenceNameTextbox.press('Enter');
+
+    // add text to editor
+    await this.editor.click();
+    await this.editor.fill('C FSW_CMD_0');
+    await this.page.getByRole('button', { name: 'Save' }).click();
+    await this.page.getByRole('button', { name: 'Close' }).click();
+
+    this.updatePage(this.page);
+    await this.tableRow.waitFor({ state: 'attached' });
+    await this.tableRow.waitFor({ state: 'visible' });
+
+    return this.sequenceName;
+  }
+
+  async deleteSequence() {
+    await this.tableRow.hover();
+    await this.page.getByRole('button', { name: 'Delete' }).click();
+    await this.page.getByRole('button', { name: 'Delete' }).click();
+  }
+
+  async goto() {
+    this.page.goto('/sequencing');
```

Comment:
This can be awaited and supplied with the optional argument of `{ waitUntil: 'networkidle' }`. That'll hopefully make this test even more flake proof by ensuring the entire page is downloaded before starting the next timeout.

Reply:
No problem. I just added it.